In pyo3 0.22 PyInt was an alias for PyLong. In pyo3 0.23 this was
switched around so PyLong was a deprecated alias for PyInt.
In pyo3 0.25 PyLong was removed.
--- rust-subversion-0.0.8.orig/src/lib.rs
+++ rust-subversion-0.0.8/src/lib.rs
@@ -147,7 +147,7 @@ impl pyo3::FromPyObject<'_> for Revision
             return Revision::from_str(&rev).map_err(|e| {
                 pyo3::exceptions::PyValueError::new_err(format!("Invalid revision: {}", e))
             });
-        } else if ob.is_instance_of::<pyo3::types::PyLong>() {
+        } else if ob.is_instance_of::<pyo3::types::PyInt>() {
             let rev = ob.extract::<i64>()?;
             return Ok(Revision::Number(Revnum::from_raw(rev).unwrap()));
         } else {
